[2022-06-27 06:24:08,624] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-27T06:19:00+00:00 [queued]>
[2022-06-27 06:24:08,635] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-27T06:19:00+00:00 [queued]>
[2022-06-27 06:24:08,636] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 06:24:08,636] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-06-27 06:24:08,636] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 06:24:08,663] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): get_datetime> on 2022-06-27 06:19:00+00:00
[2022-06-27 06:24:08,665] {taskinstance.py:1037} INFO - Dependencies not met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-27T06:19:00+00:00 [running]>, dependency 'Task Instance Not Running' FAILED: Task is in the running state
[2022-06-27 06:24:08,670] {taskinstance.py:1037} INFO - Dependencies not met for <TaskInstance: first_airflow_dag.get_datetime scheduled__2022-06-27T06:19:00+00:00 [running]>, dependency 'Task Instance State' FAILED: Task is in the 'running' state.
[2022-06-27 06:24:08,671] {local_task_job.py:99} INFO - Task is not able to be run
[2022-06-27 06:24:08,683] {standard_task_runner.py:52} INFO - Started process 94 to run task
[2022-06-27 06:24:08,705] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'scheduled__2022-06-27T06:19:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/firstdemo.py', '--cfg-path', '/tmp/tmpzb0bxblw', '--error-file', '/tmp/tmpjg5rbe4w']
[2022-06-27 06:24:08,708] {standard_task_runner.py:80} INFO - Job 18: Subtask get_datetime
[2022-06-27 06:24:08,763] {logging_mixin.py:109} INFO - Running <TaskInstance: first_***_dag.get_datetime scheduled__2022-06-27T06:19:00+00:00 [running]> on host 818d02ed7c36
[2022-06-27 06:24:08,871] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-06-27T06:19:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-27T06:19:00+00:00
[2022-06-27 06:24:08,872] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-27 06:24:08,873] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-06-27 06:24:08,901] {subprocess.py:85} INFO - Output:
[2022-06-27 06:24:08,904] {subprocess.py:89} INFO - Mon Jun 27 06:24:08 UTC 2022
[2022-06-27 06:24:08,905] {subprocess.py:93} INFO - Command exited with return code 0
[2022-06-27 06:24:08,939] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220627T061900, start_date=20220627T062408, end_date=20220627T062408
[2022-06-27 06:24:08,995] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-06-27 06:24:09,037] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
